lib/experimental/new_embed.dart

```diff
@@ -265,16 +265,19 @@ class NewEmbed {
     var hasHintOrSolution =
         context.hint.isNotEmpty || context.solution.isNotEmpty;
     showHintButton.disabled = value || !hasHintOrSolution;
+    showHintButton.hidden = !hasHintOrSolution;
```

Comment:
do we still need to disable the button to? it doesn't hurt but we can probably take that line out

Reply:
We still need to disable it while executing code (assuming it hasn't already been hidden).

Now that I'm looking at it, though, there's no reason to use hasHintOrSolution in line. Who cares if we enable a button no one can see, right?

I'll make the change, which will make this more readable.